### DISCUSSION TITLE
Next.JS + Firebase Functions

### DISCUSSION THREAD

**Discussion by douglasjkruse-work** [vercel/next.js#10672] [Help]
Hi!

Does anyone know of a solid / simple example of how to deploy functions to Firebase (that actually works as is, without a lot of whack-a-mole to figure out paths and other junk)?  It appears the examples that are currently posted in various channels are out-dated and broken.  

Thanks!

**Comment by douglasjkruse-work** [comment]
Disregard.  I got something working.  Thanks

**Reply by max** [reply]
Do you mind sharing?

**Reply by douglasjkruse-work** [reply]
ya, lame fix.  I just added a 'pages' folder (empty) to root and it finally worked.  Someone else mentioned it so I tried it even though it was not what I wanted to do.  Good luck!

**Reply by stormfar** [reply]
@douglasjkruse-work Would you mind sharing a full example or your folder structure? Having the same issue and the only examples I can find also include Firebase Hosting, which I don"t need.
